libfastpmio/io.c

```diff
@@ -939,3 +939,32 @@ write_aemit_hist(const char * filebase, const char * dataset,
     free(offset);
 }
 
+int
+read_funck(FastPMFuncK * fk, const char filename[], MPI_Comm comm)
```

Comment:
What about 'read_text_file', then call fastpm_funck_init_from_string on the caller side?